atomkraft/cli/__init__.py

```diff
@@ -97,4 +97,12 @@ def reactor(
     """
     actions = [act.strip() for act in actions.split(",")]
     variables = [var.strip() for var in variables.split(",")]
+
+    if Path(path.name).is_file():
+        print(
+            f"WARNING: Stub file already exists and it will be overwritten: {path.name}"
+        )
+        if input("Continue? [Y/n] ").lower() != "y":
+            return
```

Comment:
`[Y/n]` implies yes is the default. so an empty input should be valid.

anyway, we are using `typer`, so we can use [its prompt](https://typer.tiangolo.com/tutorial/prompt).

Reply:
I forgot about typer's prompt. Thanks! I changed the code to use it.